src/main/webapp/data/userMapData.json

```diff
@@ -41,6 +41,7 @@
       "Management Alternatives",
       "Mandurah Catholic College",
       "Methodist Ladies College",
+      "Monash College",
       "Monash University",
```

Comment:
Can check whether these two refers to the same school?

Reply:
Yes, i check out these two refer to same, and should name as "Monash University" but in the list it says "Monash College", which one i use?